testutils/config/jacoco-settings.gradle

```diff
@@ -0,0 +1,79 @@
+apply plugin: 'jacoco'
```

Comment:
Shyang, wouldn't you use a single jacoco file for this purpose? Like there https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/78

Reply:
I was just following whatever was already there.  I can update it after we merge that one.